### PR TITLE
Fix unit test defects

### PR DESCRIPTION
**Pull request by andig**
This PR requires `php-pm@dev` as interim solution until 2.0 is released. symfony-yaml was added to fix the following build error:

    1) PHPPM\Tests\SymfonyBootstrapTest::testGetRequest
    Symfony\Component\Config\Exception\LoaderLoadException: Unable to load YAML config files as the Symfony Yaml Component is not installed in /Users/andig/htdocs/php-pm-httpkernel/tests/Fixtures/Symfony/config/services.yaml (which is loaded in resource "/Users/andig/htdocs/php-pm-httpkernel/tests/Fixtures/Symfony/config/services.yaml").

    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/config/Loader/FileLoader.php:166
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/config/Loader/FileLoader.php:90
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/dependency-injection/Loader/GlobFileLoader.php:27
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/config/Loader/DelegatingLoader.php:40
    /Users/andig/htdocs/php-pm-httpkernel/tests/Fixtures/Symfony/Kernel.php:36
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:80
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/dependency-injection/Loader/ClosureLoader.php:38
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/config/Loader/DelegatingLoader.php:40
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:83
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/http-kernel/Kernel.php:653
    /Users/andig/htdocs/php-pm-httpkernel/vendor/symfony/http-kernel/Kernel.php:543
    /Users/andig/htdocs/php-pm-httpkernel/Bootstraps/Symfony.php:79
    /Users/andig/htdocs/php-pm-httpkernel/vendor/php-pm/php-pm/src/Utils.php:25
    /Users/andig/htdocs/php-pm-httpkernel/Bootstraps/Symfony.php:74
    /Users/andig/htdocs/php-pm-httpkernel/Bridges/HttpKernel.php:62
    /Users/andig/htdocs/php-pm-httpkernel/tests/SymfonyBootstrapTest.php:33

@dnna @acasademont would appreciate your feedback. Do you see any option to remove depedency on symfony/yaml?

### CODE REVIEW

**Comment by acasademont** [comment]
Mmm, this is weird. symfony-yaml is already included in symfony-bundle, shouldn't be necessary to include it manually, my new tests would have failed miserably if not. See https://github.com/symfony/framework-bundle/blob/master/composer.json

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /vendor/
-composer.lock
\ No newline at end of file
+composer.lock
+tests/Fixtures
```
Comment (acasademont) [inline .gitignore]: also, ignoring the whole fixtures directory might not be the best option, are you trying to ignore the "var" directory that Symfony creates?

Comment (andig) [reply .gitignore]: > are you trying to ignore the "var" directory that Symfony creates?

there is `cache` and other stuff as well

**Comment by andig** [comment]
> Mmm, this is weird. symfony/yaml is already included in symfony/framework-bundle, shouldn't be necessary to include it manually, my new tests would have failed miserably if not. See https://github.com/symfony/framework-bundle/blob/master/composer.json

But still failing as soon as I take it out again. On the other hand- if its required at all it shouldn't hurt if we require it ;)

**Comment by acasademont** [comment]
Still can't replicate your issue, my tests run fine on a clean master clone. "composer install" and "vendor/bin/phpunit", if not Travis would be failing.

Is "- Installing symfony/symfony (v4.2.2): Loading from cache" this line showing up when you run composer install?

**Comment by acasademont** [comment]
```
albert@AlbertWork:~/Projects/php-pm-httpkernel$[master] composer why symfony/yaml
phpunit/phpunit  5.7.27  requires  symfony/yaml (~2.1|~3.0|~4.0)
symfony/symfony  v4.2.2  replaces  symfony/yaml (self.version)
```

**Comment by acasademont** [comment]
Oh I saw it now...I had an old composer.lock file, my bad...seems like it now requires an explicit dependency.

**Comment by acasademont** [comment]
As per the cache files. I put this line on the tests

```
    public static function tearDownAfterClass()
    {
        $fs = new Filesystem();
        $fs->remove(__DIR__.'/Fixtures/Symfony/var');
    }
```

So it should clean up everything after testing. Doesn't it work for you? I can't see any leftover files

**Comment by andig** [comment]
Thanks for sharing- I wasn‘t even aware of composer why 😳

**Comment by andig** [comment]
Still wondering why we need it. Would there be a way to tell symfony to not load any yaml files?

**Pull request merged**